janusgraph-cql/src/test/java/org/janusgraph/diskstorage/cql/CassandraStorageSetup.java

```diff
@@ -49,18 +49,18 @@
 
     private static volatile Paths paths;
 
+    public static String getTestStorageHost() {
+        return System.getProperty(TEST_STORAGE_HOST);
+    }
+
     /**
      * Load cassandra.yaml and data paths from the environment or from default values if nothing is set in the environment, then delete all
      * existing data, and finally start Cassandra.
      * <p>
      * This method is idempotent. Calls after the first have no effect aside from logging statements.
      */
     public static void startCleanEmbedded() {
-        startCleanEmbedded(false);
-    }
-
-    public static void startCleanEmbedded(boolean force) {
-        if (force || System.getProperty(TEST_STORAGE_HOST) == null) {
```

Comment:
@pkendall64, @samant-orionhealth - I wasn't sure what you had in mind with the `force` option here. If using an external test server then a local instance shouldn't (and can't) be started. Is this update going to break something in your test environment?